Source/GR32.pas

```diff
@@ -1334,8 +1334,12 @@ function Color32(WinColor: TColor): TColor32; overload;
     Result := Result or TColor32(Integer(I) - 1) shl 16;
 {$ELSE}
 {$IFDEF USENATIVECODE}
-  Result := $FF shl 24 + (WinColor and $FF0000) shr 16 + (WinColor and $FF00) +
-    (WinColor and $FF) shl 16;
+  {$IFNDEF RGBA_FORMAT}

```

Comment:
This change is good but we'll need to do the same for all the other color conversion functions so they're consistent. I'll take care of that.

I will probably delete the WIN_COLOR_FIX code since I have no idea what it's doing and its use is completely undocumented.